app/assets/javascripts/discourse/app/components/reviewable-item.gjs

```diff
@@ -519,4 +543,164 @@ export default class ReviewableItem extends Component {
       return this._performConfirmed(performableAction);
     }
   }
+
+  <template>
+    <div
+      data-reviewable-id={{this.reviewable.id}}
+      class="reviewable-item {{this.customClasses}}"
+    >
+      <div class="reviewable-meta-data">
+        <span
+          class={{concatClass
+            "reviewable-type"
+            this.reviewable.humanTypeCssClass
+          }}
+        >{{this.reviewable.humanType}}</span>
+        {{#if this.reviewable.reply_count}}
+          <span class="reply-count">{{i18n
+              "review.replies"
+              count=this.reviewable.reply_count
+            }}</span>
+        {{/if}}
+        <span class="created-at">
+          <LinkTo
+            @route="review.show"
+            @model={{this.reviewable.id}}
+          >{{ageWithTooltip this.reviewable.created_at}}</LinkTo>
+        </span>
+        <span class="status">
+          {{reviewableStatus this.reviewable.status this.reviewable.type}}
+        </span>
+        <a
+          href
+          {{on "click" (fn this.explainReviewable this.reviewable)}}
+          title={{i18n "review.explain.why"}}
+          class="explain"
+        >
+          {{icon "circle-question"}}
+        </a>
+      </div>
+
+      <div class="reviewable-contents">
+        {{#if this.editing}}
+          <div class="editable-fields">
+            {{#if this.reviewable.created_by}}
+              <div class="editable-created-by">
+                {{avatar this.reviewable.created_by imageSize="tiny"}}
+                <ReviewableCreatedByName @user={{this.reviewable.created_by}} />
+              </div>
+            {{/if}}
+
+            {{#each this.reviewable.editable_fields as |f|}}
+              <div class="editable-field {{dasherize0 f.id}}">
+                {{#let
+                  (lookupComponent this (concat "reviewable-field-" f.type))
+                  as |FieldComponent|
+                }}
+                  <FieldComponent
+                    @tagName=""
+                    @value={{editableValue this.reviewable f.id}}
+                    @tagCategoryId={{this.tagCategoryId}}
+                    @valueChanged={{action "valueChanged" f.id}}
+                    @categoryChanged={{action "categoryChanged"}}
```

Comment:
removing `{{action}}` - in the same pr or in a followup?

Reply:
I think we already did it in the other pending PR. So will merge this, and then rebase that one